packages/popover/src/Popover.tsx

```diff
@@ -73,6 +81,11 @@ function Popover({
   const [contentNode, setContentNode] = useState<HTMLElement | null>(null);
   const [forceUpdateCounter, setForceUpdateCounter] = useState(0);
 
+  let { portalContainer, scrollContainer } = usePopoverPortalContainer();
+
+  portalContainer = portalContainerProp || portalContainer;
```

Comment:
why || over ??

Reply:
I'd turn the question back around, and ask you why `??` would be better here?

portalContainerProp should always be an HTMLElement, null, or undefined. Since it isn't typed as a string, number, etc, there isn't a risk of coercion causing this to evaluate any differently than `??` would. If portalContainerProp is something other than what it's typed to be, neither operator is likely to evaluate differently. This operator in marginal cases may even be more likely to be correct when the incorrect type is passed here.

Example:
```
const portalContainerProp = false;
portalContainer = portalContainerProp || portalContainer; // Equals portalContainer
portalContainer = portalContainerProp ?? portalContainer; // Equals false
```

In the above unlikely case, `||` leads to the appropriate value while `??` does not. Granted they're both incorrect if the value is `true`.

Another example:
```
const portalContainerProp = 0;
portalContainer = portalContainerProp || portalContainer; // Equals portalContainer
portalContainer = portalContainerProp ?? portalContainer; // Equals 0
```

Both examples should never happen, and ultimately the best case would be for this to throw an error. If we're thinking practically here, using `??` or `||` shouldn't actually matter in this case.

Reply:
wfm